index.js

```diff
@@ -160,6 +160,12 @@ Hbs.prototype.createRenderer = function() {
     var tplPath = hbs.getTemplatePath(tpl),
       template, rawTemplate, layoutTemplate;
 
+    if(tplPath === undefined) {
+      this.status = 500;
```

Comment:
this isn't a pattern I'm familiar with koa-hbs using. personally think it'd be better to `throw` here than anything. `status = 500` doesn't really mean anything, does it? only thing that comes to mind is an http 500 error, which wouldn't be 'not found'.